src/applications/disability-benefits/disability-rating-calculator/components/CalculatedDisabilityRating.jsx

```diff
@@ -12,12 +12,20 @@ class CalculatedDisabilityRating extends React.Component {
   }
 
   componentDidUpdate(previousProps) {
-    if (previousProps.calculatedRating !== this.props.calculatedRating) {
+    const formWasSubmitted =
+      this.props.calculatedRating &&
+      previousProps.calculatedRating !== this.props.calculatedRating;
+
+    if (formWasSubmitted) {
       focusElement(this.resultsRef.current);
     }
   }
```

Comment:
The calculated rating is focused after the form is submitted because we want screen readers to read the value. However, we don't it to focus on the value if we just replaced the value with `--`, so now we check for a `null` value for `calculatedRating`.